metricbeat/module/elasticsearch/ml_job/_meta/fields.yml

```diff
@@ -12,11 +12,114 @@
       type: keyword
       description: >
         Job state.
-    - name: data_counts.processed_record_count
-      type: long
-      description: >
-        Processed data events.
-    - name: data_counts.invalid_date_count
+    - name: model_size
+      type: group
+      fields:
+        - name: result_type
+          type: keyword
+          description: >
+            TODO
+        - name: model.bytes
+          type: long
+          description: >
+            TODO
+        - name: bucket_allocation_failures.count
+          type: long
+          description: >
+            TODO
+        - name: memory_status
+          type: long
+          description: >
+            TODO
+        - name: log_time.ms
+          type: long
+          description: >
+            TODO
+        - name: timestamp.ms
+          type: long
+          description: >
+            TODO
+        - name: total.field
+          type: group
+          fields:
+            - name: by.count
+              type: long
+              description: >
+                TODO
+            - name: over.count
+              type: long
+              description: >
+                TODO
+            - name: partition.count
+              type: long
+              description: >
+                TODO
+    - name: data.counts
+      type: group
+      fields:
+        - name: field.processed.count
+          type: long
+          description: >
+            TODO
+        - name: input.bytes
+          type: long
+          description: >
+            TODO
+        - name: input.field.count
+          type: long
+          description: >
+            TODO
+        - name: missing_field.count
+          type: long
+          description: >
+            TODO
+        - name: out_of_order.timestamp.count
+          type: long
+          description: >
+            TODO
+        - name: invalid_date.count
+          type: long
+          description: >
+            TODO
+        - name: last_data_time
+          type: long
+          description: >
+            TODO
+        - name: bucket
+          type: group
+          fields:
+            - name: empty.count
+              type: long
+              description: >
+                TODO
+            - name: sparse.count
+              type: long
+              description: >
+                TODO
+            - name: count
+              type: long
+              description: >
+                TODO
+        - name: record
+          type: group
+          fields:
+            - name: processed.count
+              type: long
+              description: >
+                Processed data events.
+            - name: earliest.ms
+              type: long
+              description: >
+                TODO
+            - name: latest.ms
+              type: long
+              description: >
+                TODO
+            - name: input.count
+              type: long
+              description: >
+                TODO
```

Comment:
Sorry if I've asked this before but, if so, I can't remember what you said before: these TODOs will show up in the docs on elastic.co so it would be good to replace them with actual content before these changes are merged into `master`. Do you want to do this now or do you plan to do this in a follow up PR before the changes are merged from the feature branch to master?

Reply:
Thank you for spotting this too. Yeah it's better to remove them, I just forgot it. I have removed the TODO and we will leave the opportunity to add them in the future.